homeassistant/components/simplisafe/__init__.py

```diff
@@ -394,6 +413,7 @@ def __init__(self, hass, api, config_entry):
         self._emergency_refresh_token_used = False
         self._hass = hass
         self._system_notifications = {}
+        self.options = config_entry.options
```

Comment:
This might not exist.

```suggestion
        self.options = config_entry.options or {}
```